oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -110,14 +154,17 @@ Examples
    * How can we take advantage of the alerts that Github provides for security prioritization?
 #. Iterations on the security process.
 #. Review industry best practices that we should consider implementing.
+#. Running an annual security survey.
+
 
 Consequences
 ************
 
+
 Security Working Group
 ======================
 
-Invite Only
+Invite only
```

Comment:
```suggestion
Invite Only
```
Shouldn't titles be fully capitalized?  Using [this](https://www.grammarly.com/blog/capitalization-in-the-titles/) as my guide.

Reply:
Yes, was imitating the first title `Reports come into security@openedx.org`. Amended.